cloud/pkg/cloudhub/channelq/channelq.go

```diff
@@ -134,6 +142,15 @@ func (q *ChannelMessageQueue) addMessageToQueue(nodeID string, msg *beehiveModel
 	nodeQueue.Add(messageKey)
 }
 
+func IsExist(source []string, msgSource string) bool {
+	for _, s := range source {
+		if s == msgSource {
+			return true
+		}
+	}
+	return false
+}
+
```

Comment:
In many places are used, it is recommended to put under `cloud/pkg/common/util/` or `/pkg/util/`